src/llmq/quorums_chainlocks.cpp

```diff
@@ -104,6 +104,14 @@ void CChainLocksHandler::ProcessNewChainLock(NodeId from, const llmq::CChainLock
     {
         LOCK2(cs_main, cs);
 
+        if (InternalHasConflictingChainLock(clsig.nHeight, clsig.blockHash)) {
```

Comment:
Should this increase misbehaving score?

Reply:
Shouldn't it be fairly easy to tell if it's a valid but old CLSIG or malicious?

Reply:
Not really. Let's say there are 2 sigs for the same height, but both for different blocks. Node A receives S1 first, node B receives S2 first. Both will now propagate their sig to the other node, resulting in BOTH nodes believing that the other one is malicious. In such cases, it's better to not increase DoS score.

Reply:
But there should never be two different CLSIGs for the same height... unless there is a fork eight blocks back (since that is what the quorum is based on)

Reply:
I mean, if it would take a fork of over 12 hours to get a conflicting CLSIG I think it is safe to assume anyone who sends one is malicious (or you are messed up locally)